doc/nrf/releases/release-notes-latest.rst

```diff
@@ -363,6 +373,12 @@ The following list summarizes the most important changes inherited from upstream
 
     * Added multiple ``nrfx_power``-related fixes to reduce power consumption.
 
+  * PWM:
+
+    * Changed GPIO configuration to Nordic HAL which allows support for GPIO  pins above 31.
+    * Added check if PWM period does not exceed 16-bit value to prevent broken behaviour.
+    * Changed PWM DT configuration to use timer phandle instead of previously used timer-instance.
```

Comment:
```suggestion
    * Changed the GPIO configuration to use Nordic HAL, which allows support for GPIO pins above 31.
    * Added a check to ensure that the PWM period does not exceed a 16-bit value to prevent erroneous behavior.
    * Changed the PWM DT configuration to use a timer phandle instead of the previously used timer instance.
```